test/SIL.XForge.Identity.Tests/Controllers/Account/AccountControllerTests.cs

```diff
@@ -62,6 +65,46 @@ public async Task Login_IncorrectPassword()
             await env.Events.Received().RaiseAsync(Arg.Any<UserLoginFailureEvent>());
         }
 
+       [Test]
+        public async Task ForgotPassword_CorrectUsernameOrEmail()
+        {
+            var env = new TestEnvironment();
+
+            var model = new ForgotPasswordViewModel
+            {
+                UsernameOrEmail = "user",
+                EnableErrorMessage = false
+            };
+            IActionResult result = await env.Controller.ForgotPassword(model);
+
+            Assert.That(result, Is.TypeOf<ViewResult>());
+            Assert.IsTrue(model.EnableErrorMessage == false);
+
+            UserEntity user = await env.Users.Query().SingleOrDefaultAsync(x => x.Username == model.UsernameOrEmail);
+            Assert.True(user.ResetPasswordKey != "jGc6Qe4i1kgM+aA4LVczTJwfHx2YuDR/", "ResetPasswordKey not saved.");
+            Assert.True(user.ResetPasswordExpirationDate != default(DateTime), "ResetPasswordExpirationDate not saved.");
+
+            string emailId = "abc@fakegmail.com";
+            string subject = "Scripture Forge Forgotten Password Verification";
```

Comment:
Did you intend to check `subject` and `body`? You specified the variables, but you don't check them.